v1/examples/cypress-grep/cypress/e2e/examples/actions.cy.js

```diff
@@ -0,0 +1,10 @@
+context('Actions', () => {
+  beforeEach(() => {
+    cy.visit('https://example.cypress.io/commands/actions')
+  })
+  it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
+    // https://on.cypress.io/type
+    cy.get('.action-email')
+        .type('fake@email.com').should('have.value', Cypress.env('grepTags'))
```

Comment:
That looks like a mistake?

```suggestion
        .type('fake@email.com').should('have.value', 'fake@email.com')
```